chapters/iterative-search.qmd

```diff
@@ -116,6 +116,7 @@ reg_mtr <- metric_set(rmse)
 #| fig-height: 5
 #| out-width: 50%
 #| fig-cap: "A visualization of model performance when only the SVM cost and scaling factor parameters are optimized. It shows the combination with the smallest RMSE and a ridge of candidate values with nearly equivalent performance."
+#| fig-altalt: "Heatmap showing optimization results for SVM parameters. A dark red diagonal ridge indicates optimal performance zones where cost and scaling factors are inversely related. The brightest point marks the lowest RMSE value at high scaling factor and low cost."
```

Comment:
I'm not yet sure if `fig-alt` works with the fenced method for making figures or with shinylive.